proposal/main.tex

```diff
@@ -19,15 +19,25 @@
 \maketitle
 
 \section*{General Problem Description}
-Description goes here...
+Aphids are among the most destructive insect pests, and monitoring aphid populations provides important data related to integrated pest management. Inefficient and labor-intensive is the manual identification of aphids. In order to reduce aphid crop damage, recent advances in computer vision algorithms - including object detection - can improve the process of identifying, classifying, and monitoring aphids.
 
 \section*{Required Implementation}
-Implementation stuff goes here...
+For the purpose of identifying the aphids, we will use YOLO/RCNN with preprocessing (fracturing the images and choosing the most likely fragments based on the percentage of green pixels). Our work on improving the baseline results includes adding our own modifications (for example, adding more contrast to the colors in the image) to further enhance the results. We will then evaluate our modified algorithms based on the chosen evaluation metrics (described in the following section). We will simplify the problem by converting it to binary: 1 for aphids and 0 for no aphids.
```

Comment:
```suggestion
To identify the aphids, we will use YOLO/RCNN with preprocessing (fracturing the images and choosing the most likely fragments based on the percentage of green pixels). Our work on improving the baseline results includes adding our modifications (for example, adding more contrast to the images' colors) to enhance the results further. We will then evaluate our modified algorithms based on the chosen evaluation metrics (described in the following section). We will simplify the problem by converting it to binary: 1 for aphids and 0 for no aphids.
```
Do we want to reduce the problem to binary classification? Feels like it makes the whole YOLO RCNN idea kind of irrelevant.

Reply:
YOLO is an algorithm that detects objects in a picture, It's not a vision course and make it a binary problem will make the process of building the model much easier